.github/workflows/markdown-linting.yml

```
@@ -12,4 +12,5 @@ jobs:
       - name: MarkdownLint mdl Action
         uses: actionshub/markdownlint@2.0.2
         with:
-          filesToIgnoreRegex: "docs/demos/.*"
+          # filter out version-specific/* for now, until we fix all issues that mdl trips over
+          filesToIgnoreRegex: "docs/demos/.*|docs/version-specific/.*"
```

```suggestion
          filesToIgnoreRegex: "(docs/demos/.*|docs/version-specific/.*)"
```

We might need to escape the / as \/